scripts/preload.js

```diff
@@ -0,0 +1,10 @@
+/*
+ * preload.js
```

Comment:
What's the purpose of this script?

Reply:
You can `node -r ./scripts/preload.js` and it drops you in the REPL with gi, gtk and gdk loaded. Maybe it's just me but I do it so often when testing suff out.

Reply:
Ok. Maybe you could leave such kind a comment somewhere (in the "for maintainers" section?).

Reply:
Sure, it has been added